### PR TITLE
Standardizes colors, header

### LINKED ISSUE #31: Add ipfs logo and make IPFS link to ipfs.io

### LINKED ISSUE #254: Architecture matrix is ugly
Like, absolutely horrible!
2019-08-12-221444_1186x400_scrot

### PR DESCRIPTION
**Pull request by jessicaschilling**
- Adds IPFS logo (link) to header
- Standardizes colors and fonts
- Minor visual tweaks

Closes #31
Closes #254 

Note: This is more of a "triage" fix than full CSS refactoring of the dist site; the intent is to take something that was pretty much unbranded and at least bring it into line with base colors, fonts etc. There are plenty of visual fixes that weren't made, but hopefully nothing is *worse*.

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@dignifiedquire -- took a stab at an old issue. Can you please have a look and recommend any additional reviewers who should examine? Thank you!

File: site/layouts/_default/single.html [hunk 1]
```diff
@@ -7,16 +7,17 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="Description">
     <link rel="stylesheet" href="site.css">
-    <link href="https://code.cdn.mozilla.net/fonts/fira.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500&display=swap" rel="stylesheet">
```
Comment (Stebalien) [inline site/layouts/_default/single.html:10]: This and the linked fonts should all be checked into the repo so this works entirely over IPFS without external resources.

**Comment by Stebalien** [comment]
The jq dependency is breaking the build on my machine (transitive dependency contextify doesn't build).

**Comment by Stebalien** [comment]
The jq you're installing is not the jq you think you're installing.

**Comment by jessicaschilling** [comment]
@Stebalien Thanks for the reminder, totally forgot to uninstall that.

**Comment by Stebalien** [comment]
`make publish`:

```
> dist.ipfs.io@1.0.0 build /home/steb/src/github.com/ipfs/distributions
> npm run dist && npm run build:assets && hugo -s site -d ../releases


> dist.ipfs.io@1.0.0 dist /home/steb/src/github.com/ipfs/distributions
> node scripts/dist.js

[Error: ENOENT: no such file or directory, open '/home/steb/src/github.com/ipfs/distributions/dists/img/current'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/steb/src/github.com/ipfs/distributions/dists/img/current'
}
```

**Comment by Stebalien** [comment]
Fixed.

**Comment by Stebalien** [comment]
Oh my, that looks SO much better!

**Comment by jessicaschilling** [comment]
@hsanjuan Do you mind a quick look, at @Stebalien's recommendation? Thanks!

**Comment by Stebalien** [comment]
The menu offsets aren't quite right. When I scroll to the bottom, it still doesn't focus ipget. It should probably switch focus once both the header of the previous section scrolls off the screen (no idea how to actually implement that).

**Comment by momack2** [comment]
Hmm, what I see is that jumping to "ipfs-update" actually just focuses ipget.

Could we also tighten the padding on mobile:

**Comment by jessicaschilling** [comment]
- brightens hover states
- bigger button text
- tightens mobile layout

cc @Stebalien @momack2

**Comment by jessicaschilling** [comment]
See: https://gateway.ipfs.io/ipfs/QmRREYVtsPWsjhyxVVydtW9xe9s8NBw2BFfrbNyHcYZGUb/

**Pull request merged**